pareto/utilities/get_data.py

```diff
@@ -20,6 +20,8 @@
 import pandas as pd
 import requests
 import numpy as np
+import openpyxl as xl
```

Comment:
```suggestion
```
This import is unused - can it be removed, or is there a reason it should be kept?

Reply:
Yes. I was using this before just sticking with pandas. I will remove this from the file as this is unused.